add model Filters_services

Nolfneo opened the pull request:
add methods:
- getFiltersServices
- insertFiltersServices
- updateFiltersServices
- deleteFiltersServices

Satuyan commented on application/models/filters_model.php:
```diff
@@ -95,4 +95,21 @@ public function deleteFilters($params = array()) {
+        }
+        $id = (int) $params['id'];
+        $this->db->where('id', $id);
+        $result = $this->db->delete('filters');
```
название таблицы не корректное

daarma replied:
Шо ита

Nolfneo committed:
add model Filters_services
add methods:
 - getFiltersServices
 - insertFiltersServices
 - updateFiltersServices
 - deleteFiltersServices

Nolfneo committed:
add model Filters_services
add methods:
 - getFiltersServices
 - insertFiltersServices
 - updateFiltersServices
 - deleteFiltersServices

Nolfneo committed:
add model Filters_services
add methods:
 - getFiltersServices
 - insertFiltersServices
 - updateFiltersServices
 - deleteFiltersServices

Nolfneo committed:
add model Filters_services
add methods:
 - getFiltersServices
 - insertFiltersServices
 - updateFiltersServices
 - deleteFiltersServices
